ward/terminal.py

```diff
@@ -548,92 +559,167 @@ def outcome_to_colour(outcome: TestOutcome) -> str:
 
 
 def scope_to_colour(scope: Scope) -> str:
-    return {Scope.Test: "green", Scope.Module: "blue", Scope.Global: "magenta",}[scope]
+    return {Scope.Test: "green", Scope.Module: "blue", Scope.Global: "magenta", }[scope]
 
 
 def output_fixtures(
-    show_scopes: bool,
-    show_docstrings: bool,
-    show_direct_dependencies: bool,
-    show_dependency_trees: bool,
+        suite,
+        show_scopes: bool,
+        show_docstrings: bool,
+        show_dependencies: bool,
+        show_dependency_trees: bool,
 ):
-    fixtures = [Fixture(f) for f in _FIXTURES]
+    fixtures = [Fixture(f) for f in _DEFINED_FIXTURES]
+
+    # we look at suite.tests so we can see the tests before they are parameterised
+    test_to_fixtures = {test: test.resolver.fixtures for test in suite.tests}
```

Comment:
We can find the parameterised instances of the tests using `test.get_parameterised_instances()`, which will expand a single test into multiple tests based. I think we want to do this here otherwise we won't find fixtures that are used inside `each`.

Reply:
Ack... I did this to dodge having to de-duplicate information during display later (I originally did use `get_parameterised_instances()`), but you're absolutely right. I'll think about how to do that nicely.